.github/workflows/test-nonroot.yml

```diff
@@ -0,0 +1,17 @@
+name: Test Nonroot
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: "1"
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build image
+      run: docker build --target nonroot -t test .
```

Comment:
So we do run multiple `docker builds` within one test run but do not use caching. This might wasting time.

Reply:
These run in parallel, so shouldn't really matter. But I guess we could run them serial and reuse the cache. In the end I don't think it will be faster...